scripts/process_logs/example_track_req.yml

```diff
@@ -0,0 +1,51 @@
+
+input_logs:                                       # List of input logs
+  - path: .                                       # Where to look for log files
+    recursive: yes                                # Whether to descend into subdirectories recursively
+    pattern: (Node\d+)\.log                       # Log file regex pattern to look for
+    node_group: 1                                 # Group number that matches node identifier
+    only_timestamped: yes                         # Whether to discard non-timestamped lines
+
+
+chains:                                           # List of processing chains
+  main:                                           # Main processing chain
+  - match: [replica: master]
+  - tag_requests:
+  - match: [tag: REQUEST]
+  - log line: output
+  - stat_received:
+  - stat_already_processed:
+  - stat_ordered:
+
+  stat_received:
+  - match: [tag: RECEIVED]
```

Comment:
How can we specify `reqId` here?

Reply:
```
- match:
  - attribute: { reqId: value }
```

Reply:
It would be great to put this line to the example config